app/models/comment.rb

```diff
@@ -0,0 +1,9 @@
+class Comment < ApplicationRecord
```

Comment:
- [Optional] Please remove these duplicate migration tables they present errors when I try to run `rails db: migrate`
![image](https://user-images.githubusercontent.com/74744787/214930624-056c90f1-818a-42ee-971d-db846aa489a2.png)

![image](https://user-images.githubusercontent.com/74744787/214930394-45675e1a-2391-4b12-ab72-9de98fdbac63.png)